superstyl/preproc/features_extract.py

```diff
@@ -1,9 +1,13 @@
+# first line only necessary for older Python versions
 from builtins import sum
 from collections import Counter
 import nltk.tokenize
 import nltk
 import regex as re
 
+#the download is a one time operation: should be placed elsewhere but I put it here for now
+nltk.download('averaged_perceptron_tagger')
```

Comment:
Better to put it inside the `if feats == 'pos'`, and then do something like,

````python
try:
    nltk.data.find("mystuff")

except:
    nltk.download("mystuff")
````
cf. https://stackoverflow.com/questions/23704510/how-do-i-test-whether-an-nltk-resource-is-already-installed-on-the-machine-runni

and 

https://stackoverflow.com/questions/30854060/if-a-block-of-code-creates-an-error-do-x-if-not-do-y-python

Reply:
yep, better practice you are right - i was unhappy with my version